src/pages/gateway/headers.md

```diff
@@ -113,3 +123,33 @@ Mesh owners can use the `responseConfig.headers` object to add response headers.
       }
     }
 ```
+
+#### CORS headers
+
+Cross-origin resource sharing or CORS allows you to pass usually restricted resources to an outside domain. Refer to [MDN's documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for more information on CORS headers.
+
+To add CORS headers to your mesh, create a `CORS` object in the `responseConfig` object, using the following key value pairs:
+
+-  `origin` (Required) - the domain of the resource you want to allow to make a CORS request
+-  `methods` (Required) - the HTTP verbs allowed in the CORS request
```

Comment:
They're not all verbs, such as OPTIONS. I went to the MDN docs, and I'm inferring  that the list of possible values are in the left nav. Consider changing the description to the following. You might want to have some different request methods than what's in the example.

```suggestion
-  `methods` (Required) - the HTTP request methods allowed in the CORS request, such as GET, POST, and OPTIONS
```